From 58cb86a73af85ed7be12f6f5edc56bdcb76f9d4a Mon Sep 17 00:00:00 2001
From: Medicine Yeh <medicinehy@gmail.com>
Date: Fri, 3 Nov 2017 12:20:43 +0800
Subject: [PATCH 2/2] Fix _DEFAULT_SOURCE redefined in Xilinx QEMU

---
 hw/core/remote-port-proto.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hw/core/remote-port-proto.c b/hw/core/remote-port-proto.c
index 4fc29ed..a2c7946 100644
--- a/hw/core/remote-port-proto.c
+++ b/hw/core/remote-port-proto.c
@@ -23,7 +23,9 @@
  * THE SOFTWARE.
  */
 
+#ifndef _DEFAULT_SOURCE
 #define _DEFAULT_SOURCE
+#endif
 #include <stdint.h>
 #include <stdlib.h>
 #include <stdio.h>
-- 
2.14.2

